test/wdio/slot-html/cmp.test.tsx

```diff
@@ -0,0 +1,186 @@
+            <content-start slot="start">start slot 2</content-start>
+          </slot-html>
+
+          <slot-html className="results5">
+            <content-start slot="start">start slot 1</content-start>
+            <content-start slot="start">start slot 2</content-start>
+            default text node
+          </slot-html>
+
+          <slot-html className="results6">
+            <content-start slot="start">start slot 1</content-start>
+            <content-default>default slot 1</content-default>
+            <content-start slot="start">start slot 2</content-start>
+            <content-default>default slot 2</content-default>
+          </slot-html>
+
+          <slot-html className="results7">
+            <content-default>default slot 1</content-default>
+            <content-start slot="start">start slot 1</content-start>
+            <content-start slot="start">start slot 2</content-start>
+            <content-default>default slot 2</content-default>
+          </slot-html>
+
+          <slot-html className="results8">
+            <content-end slot="end">end slot 1</content-end>
+            <content-end slot="end">end slot 2</content-end>
+          </slot-html>
+
+          <slot-html className="results9">
+            <content-default>default slot 1</content-default>
+            <content-end slot="end">end slot 1</content-end>
+            <content-end slot="end">end slot 2</content-end>
+            <content-default>default slot 2</content-default>
+          </slot-html>
+
+          <slot-html className="results10">
+            <content-default>default slot 1</content-default>
+            <content-default>default slot 2</content-default>
+            <content-end slot="end">end slot 1</content-end>
+            default slot text node
+            <content-end slot="end">end slot 2</content-end>
+          </slot-html>
+
+          <slot-html className="results11">
+            <content-default>default slot 1</content-default>
+            <content-end slot="end">end slot 1</content-end>
+            <content-start slot="start">start slot 1</content-start>
+            <content-end slot="end">end slot 2</content-end>
+            <content-default>default slot 2</content-default>
+            <content-start slot="start">start slot 2</content-start>
+            default slot text node
+          </slot-html>
+
+          <slot-html className="results12">
+            default slot text node
+            <content-end slot="end">end slot 1</content-end>
+            <content-start slot="start">start slot 1</content-start>
+            <content-end slot="end">end slot 2</content-end>
+            <content-default>default slot 1</content-default>
+            <content-default>default slot 2</content-default>
+            <content-start slot="start">start slot 2</content-start>
+          </slot-html>
+        </>
+      ),
+    });
+
+    await $('.results1').waitForExist();
```

Comment:
Since our first assertion is an async one we don't need this. WebdriverIO validates element existence as part of the text assertion
```suggestion
```